docs/UserGuide.md

```diff
@@ -70,13 +70,19 @@ The result of executing the input command will be displayed the lower-most box.
 
 ## Features
 
+### Command Format
+
+**Format:** `command_word Prefix/PARAMETER`
+
+* `command_word` is a word at the start of the command to specify the action to be done. <br> e.g. in `add n/NAME`, `add` is the command word.
+* `PARAMETER` are word(s) in `UPPER_CASE` to be supplied by the user.<br> e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`
+* `Prefix` is a letter before `PARAMETER` to denote the information supplied as parameter. <br>e.g. in `add n/NAME`, `n/` is a prefix to indicate that NAME is being supplied.
```

Comment:
Prefix can also be 2 letters. Better to modify this

Reply:
Good catch, will modify